src/main/java/racingCar/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingCar;
+
+public class Car {
+	private String name;
```

Comment:
Car 객체의 `name` 상태 값은 한번 지정되면 변경될 일이 없을 것 같아요. 👀
이런 경우에는 한번 할당하면 변경할 수 없도록 `final` keyword를 추가해주면 좋을 것 같네요. 

```suggestion
	private final String name;
```